modules/synapse_stage.nf

```diff
@@ -11,10 +11,10 @@ process SYNAPSE_STAGE {
     val input_folder_name
 
     output:
-    path "${input_folder_name}/"
+    stdout emit: staged_output
 
     script:
-    """    
-    synapse get -r --downloadLocation \$PWD/${input_folder_name} ${input_id}
+    """
+    synapse_stage.py \$PWD/${input_folder_name} ${input_id}
```

Comment:
I'm thinking a bit more about the extensions folder we want to add to the synapsepythonclient.

Suppose the extensions had a `synapse_stage.py`. Do you know if you would be able to call python scripts directly in this `.nf` file from the installed dependency? The idea being that stage seems like a "general" extension that could be maintained in the synapsepythonclient, and, it would remove you needing to continue having it here.

Reply:
Cc @BWMac

Reply:
It is possible, and that's already what we're planning on doing for the `VALIDATE` and `SCORE` steps in this workflow: Users will provide us with a container that houses their scripts, and the calls to the scripts would basically look like these calls to our internal `bin/` scripts, only the scripts themselves are houses somewhere else (in the specified container).